kuma/wiki/migrations/0011_create_spam_data.py

```diff
@@ -4,24 +4,14 @@
 from django.conf import settings
 from django.db import migrations
 from kuma.spam.constants import SPAM_CHECKS_FLAG
-from kuma.wiki.constants import SPAM_EXEMPTED_FLAG
 
 
 def create_trusted_writers(apps, schema_editor):
     # first create a new trusted writers user group
     Group = apps.get_model('auth', 'Group')
     group, created = Group.objects.get_or_create(name='Trusted writers')
 
-    # then add that user group to the list of people exempted from spam checks
     Flag = apps.get_model('waffle', 'Flag')
-    exempted_flag, created = Flag.objects.get_or_create(
-        name=SPAM_EXEMPTED_FLAG,
-        defaults={
-            'note': 'Used to decide which users are exempted from spam checks.'
-        },
-    )
-    exempted_flag.groups.add(group)
-
```

Comment:
I shouldn't have merged this... We try not to change past migrations except in extreme circumstances.

Reply:
@jwhitlock Ah ok, will remember for future changes. Just to double check. Did you add this code back and merge it to master?

Reply:
Yes, I added it in a second commit to #4553.

I hope to squash migrations in December, and we'll have less of them to cause us problems 😄